### PR TITLE
DataOwner and 3rdPartyDataowner are in separate blocks

### PR DESCRIPTION
**Pull request by procrastinatio**
[Demo](https://mf-chsdi3.int.bgdi.ch/shorten/772fb6aa4b)

Refs: https://github.com/geoadmin/mf-geoadmin3/issues/4032

Should be deployed alongs with: https://github.com/geoadmin/service-print/pull/61 (no break, but ugly comma at end of strings)

### CODE REVIEW

**Comment by davidoesch** [comment]
do align all copyrights to the right
image

**Comment by procrastinatio** [comment]
Updated. Mapfish Print is no wordprocessor, changing color is only possible within a block.

**Comment by procrastinatio** [comment]
All data owers on the same line, in black rechts bündig

screenshot

**Comment by davidoesch** [comment]
Me happy

File: src/components/print/PrintDirective.js [hunk 1]
```diff
@@ -387,8 +379,8 @@ goog.require('ga_urlutils_service');
                   display: [$scope.layout.map.width, $scope.layout.map.height],
                   // scale has to be one of the advertise by the print server
                   scale: $scope.scale.value,
-                  dataOwner: dataOwner,
-                  thirdPartyDataOwner: thirdPartyDataOwner,
+                  dataOwner: allDataOwner,
+                  thirdPartyDataOwner: false,
```
Comment (oterral) [inline src/components/print/PrintDirective.js]: can we remove it or it breaks the print ?

Comment (procrastinatio) [reply src/components/print/PrintDirective.js]: Yes, me may. I does not break

**Pull request merged**